core/src/main/java/nu/yona/server/CoreConfiguration.java

```diff
@@ -75,7 +75,8 @@ RepositoryProvider repositoryProvider()
 	public ReloadableResourceBundleMessageSource messageSource()
 	{
 		ReloadableResourceBundleMessageSource messageBundle = new ReloadableResourceBundleMessageSource();
-
+		
+		messageBundle.setFallbackToSystemLocale(false);
```

Comment:
Good point. I can take it out and do it in a different PR. The issue is that the tests didn't run on my home PC, because it used Dutch as system language.